### PR TITLE
automount disks script, grub bootloader themes script, bottles install script, ssh commands automation 

### PR DESCRIPTION
**Pull request by guruswarupa**
# Pull Request

## Title
- Automount disks script added.
- grub bootloader theme script added.
- bottles install script added.
- few ssh commands automation such as key generation, share key to remote host, disabling password auth, run a command on a remote host, and many more useful commands.

### CODE REVIEW

File: src/commands/system-setup/9-bottles-setup.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    esac
+    fi
+    flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+    flatpak install -y flathub com.usebottles.bottles
+    printf "%b\n" "${GREEN}Bottles installed successfully. Restart the system to apply changes...${RC}"
+}
+
+checkEnv
+checkEscalationTool
+clear
```
Comment (nnyyxxxx) [inline src/commands/system-setup/9-bottles-setup.sh:36]: ``clear``

Needs to be removed.

File: src/commands/system-setup/8-grub-theme.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+themeinstall(){
+    cd "$HOME" && git clone "https://github.com/ChrisTitusTech/Top-5-Bootloader-Themes"
+    cd "Top-5-Bootloader-Themes"
+    $ESCALATION_TOOL ./install.sh
+}
+
+checkEnv
+checkEscalationTool
+clear
```
Comment (nnyyxxxx) [inline src/commands/system-setup/8-grub-theme.sh:12]: ``clear``

Needs to be removed.

File: src/commands/system-setup/7-automount.sh [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        echo "Drive mounted successfully at $mount_point."
+    else
+        echo "Failed to mount the drive."
+        exit 1
+    fi
+}
+
+checkEnv
+checkEscalationTool
+clear
```
Comment (nnyyxxxx) [inline src/commands/system-setup/7-automount.sh:83]: ``clear``

Needs to be removed.

File: src/commands/system-setup/9-bottles-setup.sh (hunk 1 above)
Comment (guruswarupa) [reply src/commands/system-setup/9-bottles-setup.sh:36]: Clear command is given to make it more interactive and not show the output which is getting printed by the checkescaltion and other commands.

**Comment by nnyyxxxx** [comment]
@guruswarupa Too many scripts are being added here; you should move that ssh script to a new PR so this PR is easier to review.

**Comment by guruswarupa** [comment]
> @guruswarupa Too many scripts are being added here; you should move that ssh script to a new PR so this PR is easier to review.

only 4 :) not adding anymore to this.

File: src/commands/utils/ssh.sh [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+enable_password_auth() {
+    echo "Disabling SSH password authentication and enabling key-only login..."
+    read -p "Enter the alias of the host: " host_alias
+    echo
+    ssh $host_alias "
+        sudo -S sed -i 's/^#PasswordAuthentication no/PasswordAuthentication yes/' /etc/ssh/sshd_config &&
+        sudo -S sed -i 's/^PasswordAuthentication no/PasswordAuthentication yes/' /etc/ssh/sshd_config &&
+        sudo -S sed -i 's/^#PubkeyAuthentication yes/PubkeyAuthentication no/' /etc/ssh/sshd_config &&
+        sudo -S sed -i 's/^PubkeyAuthentication yes/PubkeyAuthentication no/' /etc/ssh/sshd_config &&
+        sudo -S systemctl restart sshd
```
Comment (nnyyxxxx) [inline src/commands/utils/ssh.sh:75]: Every sudo needs to be changed to $ESCALATION_TOOL

File: src/commands/system-setup/8-grub-theme.sh (hunk 2 above)
Comment (nnyyxxxx) [inline src/commands/system-setup/8-grub-theme.sh:12]: ``clear`` still needs to be removed out of all of these scripts.

File: src/commands/utils/ssh.sh (hunk 4 above)
Comment (guruswarupa) [reply src/commands/utils/ssh.sh:75]: sudo is being used on a different machine and not in the machine its run.  its ssh script

Comment (nnyyxxxx) [reply src/commands/utils/ssh.sh:75]: @guruswarupa It shouldn't matter.

**Comment by ChrisTitusTech** [comment]
Sorry for the inconvenience. We had a massive restructure of the codebase to improve future development. Because of this can you update your PR to the new structure. Thank you for your assistance and contribution.

**Comment by guruswarupa** [comment]
@ChrisTitusTech  @nnyyxxxx i've fixed those conflicts.

**Comment by nnyyxxxx** [comment]
> @ChrisTitusTech @nnyyxxxx i've fixed those conflicts.

Awesome!

**Comment by jeevithakannan2** [comment]
Please split this PR into separate features

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request closed without merging**